ahnlich/ai/src/server/handler.rs

```diff
@@ -574,6 +621,65 @@ impl AiService for AIProxyServer {
         }))
     }
 
+    #[tracing::instrument(skip_all)]
+    async fn convert_store_input_to_embeddings(
```

Comment:
Would need to add support for this to the rust, golang and python AI clients respectively

You've generated the types already for Rust and Python , but yet to generate for Golang

But for Rust, there's a bit of boilerplate (I should really turn that into a macro) within`ahnlich/client/src/ai.rs` to neatly wrap it up for Rust clients pipeline and non-pipeline usage

For Python I think a test would be helpful to confirm this works

For Golang, there is a Makefile in the directory to regenerate and probably also a test